custom-attachments/src/main/res/drawable/ic_calendar.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Could we replace the attachment icon with this drawable?
I think it would be instantly more obvious to the user using this sample what it does.

Reply:
Done